apps/server/src/repositories/learnroom/dashboard.repo.ts

```diff
@@ -0,0 +1,19 @@
+import { Injectable } from '@nestjs/common';
+import { DashboardEntity, DefaultGridReference, GridElement } from '../../entities/learnroom/dashboard.entity';
+
+export interface IDashboardRepo {
+	getUsersDashboard(): Promise<DashboardEntity>;
+}
+
+@Injectable()
+export class DashboardRepo implements IDashboardRepo {
+	getUsersDashboard(): Promise<DashboardEntity> {
+		const gridArray: GridElement[] = [];
+		const diagonalSize = 5;
+		const elementReference = new DefaultGridReference('exampletitle');
+		for (let i = 0; i < diagonalSize; i += 1) {
+			gridArray.push(new GridElement(Math.floor(Math.random() * 6 + 1), i + 1, elementReference));
+		}
+		return Promise.resolve(new DashboardEntity({ grid: gridArray }));
```

Comment:
is tehre an async call in this method? why using promise.resolve? how will this method load data?